src/colorizer/Dataset.ts

```diff
@@ -298,6 +334,17 @@ export default class Dataset {
     this.timesFile = manifest.times;
     this.centroidsFile = manifest.centroids;
 
+    if (manifest.backdrops) {
+      for (const { name, key, frames } of manifest.backdrops) {
+        this.backdropData.set(key, { name, frames });
+        if (frames.length !== this.frameFiles.length || 0) {
+          console.warn(
```

Comment:
wondering if this could be promoted to error.  if they are not matched then playback will have rejected promises when trying to load.